bin/here.py

```diff
@@ -20,7 +20,7 @@ def main():
         "-v {}:/work".format(cwd),
         "-w /work",
         "-it",
-        "faasm/sysroot:v{}".format(toolchain_ver),
+        "faasm/sysroot:{}".format(toolchain_ver),
```

Comment:
Containers in Docker Hub used to have a `v` before the version, but this is fixed now